From: Karel Zak <kzak@redhat.com>
Date: Mon, 10 Dec 2018 12:03:18 +0100
Subject: fstrim: fix usage()
Origin: https://git.kernel.org/pub/scm/utils/util-linux/util-linux.git/commit/?id=f78f2983d305fb18db6965661e14f73297b344b7
Bug-Debian: https://bugs.debian.org/917150

Signed-off-by: Karel Zak <kzak@redhat.com>
---
 sys-utils/fstrim.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sys-utils/fstrim.c b/sys-utils/fstrim.c
index 00652cbb1968..105ee89e789d 100644
--- a/sys-utils/fstrim.c
+++ b/sys-utils/fstrim.c
@@ -340,7 +340,7 @@ static void __attribute__((__noreturn__)) usage(void)
 	fputs(_(" -l, --length <num>  the number of bytes to discard\n"), out);
 	fputs(_(" -m, --minimum <num> the minimum extent length to discard\n"), out);
 	fputs(_(" -v, --verbose       print number of discarded bytes\n"), out);
-	fputs(_(" -d, --dry-run       does everything, but trim\n"), out);
+	fputs(_(" -n, --dry-run       does everything, but trim\n"), out);
 
 	fputs(USAGE_SEPARATOR, out);
 	printf(USAGE_HELP_OPTIONS(21));
-- 
2.20.1

