From a6a9ab62006b9bde1a79333a1fd1785706bde6db Mon Sep 17 00:00:00 2001
From: Andreas Henriksson <andreas@fatal.se>
Date: Mon, 12 Nov 2018 19:21:01 +0100
Subject: [PATCH] Add Documentation key to uuidd.service

---
 misc-utils/uuidd.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/misc-utils/uuidd.service.in b/misc-utils/uuidd.service.in
index 531672765..a43b3c3e0 100644
--- a/misc-utils/uuidd.service.in
+++ b/misc-utils/uuidd.service.in
@@ -1,5 +1,6 @@
 [Unit]
 Description=Daemon for generating UUIDs
+Documentation=man:uuidd(8)
 Requires=uuidd.socket
 
 [Service]
-- 
2.19.1

